### ISSUE TITLE
Publishing the views is not working or not documented

### ISSUE THREAD

**Issue by omarherri** [spatie/laravel-medialibrary#3567]
To customizing the rendered HTML i tried to publish the view as mentioned in the documentation using:

`php artisan vendor:publish --provider="Spatie\MediaLibrary\MediaLibraryServiceProvider" --tag="views"`

But this returned _No publishable resources for tag [views]._

**Comment by patinthehat** [comment]
@omarherri Have you tried running 
```sh
php artisan vendor:publish --provider="Spatie\MediaLibrary\MediaLibraryServiceProvider" --tag="medialibrary-views"
```
This follows the same format as the other publish commands, even though the docs indicate "views" as the correct tag.  Please let me know if you still run into an issue, and I'll dig in and get to the bottom of it! :+1:

**Comment by omarherri** [comment]
@patinthehat, I tried your solution, but I still received the same message: _"No publishable resources for tag [medialibrary-views]."_ 

So, I looked up the package and noticed that the views are declared under the 'media-library' namespace. Therefore, I modified the command to:

`php artisan vendor:publish --provider="Spatie\MediaLibrary\MediaLibraryServiceProvider" --tag="media-library-views"`
 
 And it worked for me! Thank you for pointing me in the right direction.
